lib/ops_manager_ui_drivers/wait_helper.rb

```diff
@@ -40,7 +47,7 @@ def retry_times(retries, &blk)
       retries -= 1
       Logger.debug "------- retries_left=#{retries}"
       if retries > 0
-        sleep(SLEEP_INTERVAL)
+        sleep(WaitHelper.sleep_interval)
```

Comment:
For similar reasons as above, this should not reference the module. The end result would look something like `sleep(sleep_interval)`.